homeworks/roman.strokan_Strokan/homework_3/z1.js

```diff
@@ -0,0 +1,44 @@
+
+// 1.1
+function func(a) {
+    const b = a * a;
+    // console.log(`${a}^2=${b}`);
+    return b;
+}
+// func(5);
+
+// 1.2
+function arrayCreation(a, b) {
+    const arr = [];
+    let c = 0;
+    for (let i = 0; c <= b; i++) {
+        arr[i] = c;
```

Comment:
assigning array elements by index is not the best practice... try using `push()` method

hint: you could simplify you script by using `i += a` instead of `i++`